src/utils/box.ts

```diff
@@ -259,6 +259,7 @@ export function box(text: string, _opts: BoxOpts = {}) {
   }
   // Include the title if it exists with borders
   if (opts.title) {
+    const title = _color ? _color(opts.title) : opts.title;
```

Comment:
Although I followed other part of the code base, there seems no such case where `_color` is falsy in terms of type definition of `getColor` function and its actual code as far as I checked.
If it's not intentional for some reasons, I will fix it as well.

https://github.com/unjs/consola/blob/35c9d6ea45ef60faaabb2a34cc60fdc621cf7730/src/utils/box.ts#L228-L243

Reply:
Indeed. recent implementation of `getColor` had fallback.

Feel free to make another PR to refactor